gen/tests/test_validation.py

```diff
@@ -141,6 +141,26 @@ def test_invalid_mesos_dns_set_truncate_bit():
         true_false_msg)
 
 
+def test_validate_mesos_recovery_timeout():
+    validate_success(
+        {'mesos_recovery_timeout': '24hrs'})
```

Comment:
What about '24.5hrs'? Does it have to be an integer?

Reply:
Not anymore with the latest commit.